main.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function(){
+
+$("a").click(function(event){
+    event.preventDefault();
```

Comment:
Interesting approach! in this particular assignment, this totally works. However, I would not advise doing this in the real world. The reason is, you're basically blocking ALL links (`a` tags) from following their `href` values. It just so happens in this assignment, that's fine. But in the real world, you'll likely have a mix of normal `<a href="link-to-some-website">blah</a>` that you want to behave as normal, and some that you want to just act like buttons with JavaScript.

Instead, I would suggest simply putting the `event.preventDefault()` in each of the respective event handlers: `showMore`, `showLess`, and `learnMore`.